.idea/codestyles/Project.xml

```diff
@@ -29,7 +29,16 @@
     <JetCodeStyleSettings>
       <option name="PACKAGES_TO_USE_STAR_IMPORTS">
         <value>
-          <package name="kotlinx.android.synthetic" withSubpackages="true" static="false" />
+          <package name="kotlinx.android.synthetic" alias="false" withSubpackages="true" />
```

Comment:
Did you run the code formatter over the paymentsheet packages in the test and main, just to make sure everything is in sync with this?  If there are, might make sense to do it in a separate commit.

Reply:
Reverting this, wasn't intentional to add it.